net-news/rssguard/rssguard-9999.ebuild

```diff
@@ -3,41 +3,36 @@
 
 EAPI=6
 
-inherit git-r3 qmake-utils
+inherit flag-o-matic git-r3 qmake-utils
 
 DESCRIPTION="A tiny RSS and Atom feed reader"
 HOMEPAGE="https://github.com/martinrotter/rssguard"
-EGIT_REPO_URI="https://github.com/martinrotter/${PN}.git"
+EGIT_REPO_URI="https://github.com/martinrotter/rssguard.git"
```

Comment:
Why this change?

Reply:
https://devmanual.gentoo.org/ebuild-writing/variables/
```
The examples of bad constant-value variable references are:

using ${PN} in HOMEPAGE, EGIT_REPO_URI or the domain of SRC_URI — it breaks URL parsing in editors and terminals, and makes it hard to copy the link for external use (e.g. while reviewing via gitweb), 
```

Reply:
Thanks, that I can accept.